src/_data/rest_api.json

```diff
@@ -102,6 +102,12 @@
 
         "vehicles-for-agency": {
             "description": "get active vehicles for an agency"
+        },
+        "search (stop)": {
```

Comment:
please alphabetize these two methods

Reply:
Also please rename the new pages to `search-stop` and `search-route`

Reply:
Sir, i need 2 little bit clarifications
1. either we can keep `search-route` or `Search-Route` for the new page name and method name, because the both should be same else it will show 404
2. we can change `page title` which can be differ from the `page name`
 can we keep `search-route` format everywhere?